apps/pp/views/annotations.py

```diff
@@ -118,11 +118,8 @@ def post(self, request):
                                              context={'request': request}).data)
 
     def get_queryset(self):
-        queryset = Annotation.objects \
-            .filter(active=True).annotate(
-            upvote_count=Coalesce(
-                Sum(Case(When(feedbacks__id=True, then=1)), default=0, output_field=IntegerField()),
-                0),
+        queryset = Annotation.objects.filter(active=True).annotate(
+            upvote_count=Count('feedbacks__id')
         ).prefetch_related(
```

Comment:
I remember I used this convoluted query since in some cases `Count` returned `None` (there was no `0` default). I see now it works perfectly, which might mean Count behaviour was improved with django upgrade or something else has changed. Anyway, we'd better keep eye on this...

Reply:
My story about this code is that I wrote tests which made it failing, because on SQLite it didn't actually work (like always _zero_ result ), so I gave it a try and replaced it with pure `Count` and it worked. I see PostgreSQL as more reliable, so I am pretty optimistic about it. But yeah, let's keep an eye on it.